server/jetstream_api.go

```diff
@@ -413,6 +416,13 @@ type JSApiConsumerListResponse struct {
 
 const JSApiConsumerListResponseType = "io.nats.jetstream.api.v1.consumer_list_response"
 
+// JSApiConsumerGetNextRequest is for getting next messages for pull based consumers.
+type JSApiConsumerGetNextRequest struct {
+	Expires time.Time `json:"expires,omitempty"`
```

Comment:
difficult for other languages, can we make this seconds?

Reply:
This is a common RFC for time, but yes we could. Should look into how other represent timeouts.